### ISSUE TITLE
scripts in sbin

### ISSUE THREAD

**Issue by minad** [SietsevanderMolen/i3-qubes#14]
Hi,

I would prefer if the scripts are not in /sbin but in an i3-specific directory. Otherwise I would put them in /usr/bin and name them consistently.
- /usr/bin/qubes-i3-xdg-autostart
- /usr/bin/qubes-i3-sensible-terminal
- /usr/bin/qubes-i3-status

**Comment by SietsevanderMolen** [comment]
I tried to model them after the "original" i3 versions, i3status and i3-sensible-terminal, so I think they should be in /usr/bin/ like those are.

The current naming is because I modeled them after the "original" tools.
i3-sensible-terminal -> qubes-i3-sensible-terminal
i3status -> qubes-i3status

xdg-autostart was originally not related to i3 or qubes, and useful by itself, so I didn't include i3 there.
